src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -116,8 +116,8 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
   return (
     <div className="EntityFinderDetailsView">
       <table>
-        <thead>
-          <tr className="EntityFinderDetailsView__HeaderRow">
+        <thead className={entities.length === 0 ? 'Inactive' : ''}>
```

Comment:
This class was unused. We will instead use `thead.Inactive` to set opacity of the table headers if there are no entities